test/data/metadata-values.json

```diff
@@ -26,6 +26,7 @@
   "meta-data/local-hostname": "ip-10-196-24-63.ec2.internal",
   "meta-data/instance-type": "t2.small",
   "meta-data/instance-id": "i-aaaf2d1a",
+  "meta-data/iam/security-credentials": "fake-fake",
```

Comment:
The spec is failing saying this was in expected but not actual. It may not be necessary here. Adding the expectation to the spec should be enough.